app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str = "") -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = SoftwareEngineer.skills + ["JavaScript", "HTML", "CSS"]
```

Comment:
u don't need to hardcode skills like that
just extend skills in the constructor method